tests/unit_tests/test_plots.py

```diff
@@ -142,6 +142,44 @@ def test_highlight_domains():
     plots.highlight_domains(model.geometry, mats)
 
 
+def test_material_color():
+    plot = openmc.Plot()
+    plot.color_by = 'material'
+    plots = openmc.Plots([plot])
+
+    model = openmc.examples.pwr_pin_cell()
+    same_id = 1001
+    fuel = model.geometry.get_cells_by_name("fuel")[0]
+    fuel.id = same_id
+    fuel.color = "cornflowerblue"
+    uo2 = fuel.fill
+    uo2.id = same_id
+    uo2.color = "goldenrod"
+
+    plots.colorize(model.geometry)
+    assert plot.colors.get(fuel) is None
```

Comment:
```suggestion
    assert plot.colors.get(fuel) is None
```
I see fuel.color is set to "cornflowerblue" then ends up being None. Is this because they share the same ID

Reply:
This essentially tests that when `color_by == 'material'`, that `Plot.colors` maps only by material and not cell. I'll add comments.